src/components/modules/checkbox/checkbox.jsx

```diff
@@ -1,79 +1,89 @@
 import React, { Component } from 'react';
 import classNames from 'classnames';
+import DefaultProps  from '../../defaultProps';
 
 export default class Checkbox extends Component {
     static propTypes = {
+        ...DefaultProps.propTypes,
+
+        /**
+         * State checked
+         */
         checked: React.PropTypes.bool,
-        children: React.PropTypes.node,
-        className: React.PropTypes.any,
-        component: React.PropTypes.oneOfType([
-            React.PropTypes.element,
-            React.PropTypes.string
-        ]),
-        defaultClasses: React.PropTypes.bool,
+        /**
+         * Does not allow user interaction
+         */
         disabled: React.PropTypes.bool,
-        fitted: React.PropTypes.bool,
-        indeterminate: React.PropTypes.bool,
-        name: React.PropTypes.string,
+        /**
+         * It does disabled, but does not allow user interaction
+         */
+        readOnly: React.PropTypes.bool,
+        /**
+         * Callback handler to click checkbox
+         */
         onClick: React.PropTypes.func,
+        /**
+        * Checkbox - appearance
+         */
         radio: React.PropTypes.bool,
-        readOnly: React.PropTypes.bool,
+        /**
+        * Attr name
+         */
+        name: React.PropTypes.string,
+        /**
+        * Checkbox - appearance
+         */
         slider: React.PropTypes.bool,
-        toggle: React.PropTypes.bool
+        /**
+        * Checkbox - appearance
+         */
+        toggle: React.PropTypes.bool,
```

Comment:
Now noticed another thing:
There few appearance variation. May be better make single type: ```React.PropTypes.oneOf(['default', 'radio', 'toggle', 'slider'])``` . By default it's "default" and has no className mapping.
Because it doesn't make sense if i write ```<Checkbox slider toggle />```